docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -238,3 +238,42 @@ Sentry.feedbackIntegration({
     onAddScreenshot={handleChooseImage}
 });
 ```
+
+## Theming
+
+You can also customize the Feedback Widget colors to match your app's theme. The example below shows how to customize the widget backgroung and foreground for the light and dark system themes with the `feedbackIntegration`.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.feedbackIntegration({
+  colorScheme: 'system',
+  themeLight: {
+    foreground: '#ff0000',
+    background: '#00ff00',
+  },
+  themeDark: {
+    foreground: '#00ff00',
+    background: '#ff0000',
+  },
+}),
```

Comment:
i: We could use a full init to avoid confusion on calling the integration outside of Sentry init
```suggestion
Sentry.init({
  integrations: [
    Sentry.feedbackIntegration({
      colorScheme: 'system',
      themeLight: {
        foreground: '#ff0000',
        background: '#00ff00',
      },
      themeDark: {
        foreground: '#00ff00',
        background: '#ff0000',
      },
    }),
  ],
});
```